client/tests/test_online_providers.py

```diff
@@ -0,0 +1,80 @@
+from featureform.providers import get_provider, OnlineStore, OnlineTable, LocalFileStore
+from featureform.register import ScalarType
+import pytest
+import uuid
+from datetime import datetime
+
+
+@pytest.mark.parametrize(
+    "provider",
+    ["file"],
+)
+class TestOnlineProvider:
```

Comment:
This might be a nit, but @ahmadnazeri was suggesting we move away from the class-based test pattern.

Reply:
Whats the alternative. I have nested params I want to go through